src/app/courses/progress-courses/courses-progress-leader.component.ts

```diff
@@ -71,6 +78,21 @@ export class CoursesProgressLeaderComponent implements OnInit, OnDestroy {
     });
   }
 
+  getTeams(course) {
+    const selectors = {
+      teamPlanetCode: this.selectedPlanetCode,
+      courses: { '$elemMatch': { '_id': course._id } }
+    };
+    this.couchService.findAll('teams', findDocuments(selectors, 0))
+      .pipe(tap(teams => this.courseTeams = teams))
```

Comment:
This `tap` is unnecessary.  In fact both of the `pipes` here should just be performed in one subscribe.